### ISSUE TITLE
Hardcoded expo.modules.widgets.example

### ISSUE THREAD

**Issue by zheorekhy** [gitn00b1337/expo-widgets#27]
I'm sorry but I have another issue. 
I've copied ios widget from example folder. 
My app.json has such configuration:
``` 
[
        "@bittingz/expo-widgets",
        {
          "ios": {
            "src": "./widgets/ios",
            "mode": "development",
            "moduleDependencies": [
              "MyData.swift",
              "LogHandler.swift"
            ],
            "xcode": {
              "configOverrides": {
                "SWIFT_VERSION": "5.0"
              }
            }
          }
        }
      ]
```

But I press on the button and I my widget doesn't receive a message.
I have found in your module hardcoded id: `let widgetSuite = UserDefaults(suiteName: "group.expo.modules.widgets.example.expowidgets")`

<img width="868" alt="Снимок экрана 2024-09-05 в 16 34 46" src="https://github.com/user-attachments/assets/5bc4dfc9-f365-421d-93ec-5f18bf21a542">
What to do? 

My skype: oreshkin_life

**Comment by zheorekhy** [comment]
If I change my bundleIdentifier from "com.app.xxx" to "expo.modules.widgets.example" then it works as expected

**Comment by gitn00b1337** [comment]
Please see the readme, you can provide your own module.swift and adjust the id as you wish.

**Comment by zheorekhy** [comment]
@gitn00b1337 I've reinstalled modules and now it works. It replaces the id in the module (ExpoWidgetsModule.swift) right after ios:prebuild command (I copied it from example). I don't understand what happens and what files actually are used.

> Please see the readme, you can provide your own module.swift and adjust the id as you wish.
"src": "./widgets/ios", this?

What is your contact? Maybe I will need to request widget development from you tomorrow)
